src/service/poc_mobile.proto

```diff
@@ -80,6 +83,41 @@ message subscriber_location_ingest_report_v1 {
   subscriber_location_req_v1 report = 2;
 }
 
+enum signal_level {
+  NO = 0;
+  LOW = 1;
+  MEDIUM = 2;
+  HIGH = 3;
+}
+
+// Radio signal level and power in the h3 hex.
+message radio_hex_signal_level {
+  uint64 hex_id = 1;
+  signal_level signal_level = 2;
+  double signal_power = 3;
```

Comment:
comment with type info?

Reply:
dBm, apparently

Reply:
Can we use a u32 or something like we do with IoT PoC and use ddbm? trying to avoid floats since they're massive and terrible language support